relayer/relayer.py

```diff
@@ -2,18 +2,16 @@
 from tronpy.providers import HTTPProvider
 from tronpy.keys import PrivateKey
 from web3 import Web3
-import aiohttp
 import json
 import asyncio
 from base58 import b58encode_check
-from flask import Flask, request
-from threading import Thread
+import os
 
 config = json.load(open("config.json"))
 client = Tron(HTTPProvider("https://api.trongrid.io", api_key=config["trongrid_api_key"]))
 abi = json.load(open("abi.json"))["abi"]
 
-usdt = client.get_contract("TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t")
+sunswap_v2 = client.get_contract("TXF1xDbVGdxFGbovmmmXvBGu8ZiE3Lq4mR")
```

Comment:
i'll pass this one because the python relayer is needed for testing. we'll replace it with a production one written in rust in the future. if i didn't answer any of your other comments, it means i left them as "to be done in a production relayer" in my head